pkg/entity/batch_exporter/cmd.go

```diff
@@ -40,36 +45,43 @@ func GetCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "batch-exporter [name]",
 		Short: "Get Batch exporter by name",
+		Long:  string(content),
 		PreRun: func(cmd *cobra.Command, args []string) {
 			printer = configurePrinter(cmd)
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			get(&args[0], cmd)
 		},
 		Args:              cobra.ExactArgs(1), // the stream name
+		DisableAutoGenTag: true,
 		ValidArgsFunction: namesCompletion,
 	}
 }
 func ListCmd() *cobra.Command {
 	return &cobra.Command{
 		Use:   "batch-exporters",
```

Comment:
Not every file indentation has changed. So if you've added an `.editorconfig`, please reformat all files :)